sdb/commands/zfs/dbuf.py

```diff
@@ -0,0 +1,123 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+import argparse
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class DslDir():
+
+    @staticmethod
+    def name(dd):
```

Comment:
Can you comment on the motivation for creating a class for this (and others below) method? Would it be better to just use a function, without classes?

Reply:
Originally I was thinking these classes would grow into more comprehensive utilities / commands for dsl_dir's (etc).  I'd be fine with changing these to simple helper funcs within the Dbuf class

Reply:
That's a good question and I'd be interested to hear about it too.
For a similar use-case (`slub_helpers.py`) I ended up using functions for everything instead of creating a class with methods but that was mostly due to laziness, I can definitely see the point of having specific types that mirror the actual types that are being represented (e.g. the hope of having python do some type-checking for us instead of us explicitly checking everything through `drgn`).